notebooks/5094_harmonisation.py

```diff
@@ -82,6 +78,23 @@
 
 # model_pre_processed_for_gridding
 
+# %%
+# Remove Emissions|*|CO2 AFOLU
+vars = model_pre_processed_for_gridding.index.get_level_values("variable")
+
+# Build mask to drop unwanted variables
+mask_keep = vars == "Emissions|CO2|CO2 AFOLU"
+mask_drop = vars.str.endswith("CO2 AFOLU") & ~mask_keep
+
+# Drop all other "*|CO2 AFOLU"
+idf_filtered = model_pre_processed_for_gridding[~mask_drop]
+
+# Rename the one you want to keep
+model_pre_processed_for_gridding = idf_filtered.rename(
```

Comment:
Indeed we can currently only harmonize CO2|AFOLU on the global level, not on the regional level.